copydlls.sh

```diff
@@ -274,11 +280,24 @@ makensis() {
     cd "$TARGETDIR/nsis"
     pwd
     echo "This is still a work in progress... but GPL..." > ../COPYRIGHT.txt
+    TAG=$(echo "$VERSION" | grep -Po '^\d+\.\d+\.\d+')
+    if [ -z $TAG ]; then
+      TAG="master"
+    fi
+
+    echo "Generating full installer"
+    "$MAKENSIS" \
+        //DPRODUCT_VERSION=$VERSION \
+        //DOUTFILE="..\kicad-$VERSION-$ARCH-full.exe" \
+        //DARCH="$ARCH" \
+        install.nsi
+
+    echo "Generating light installer"
     "$MAKENSIS" \
-        //DOPTION_STRING="native-mingw-with-scripting-$ARCH" \
```

Comment:
This is unrelated, to the actual change, split into separate commit

Reply:
Done